src/components/ShortTextInput.jsx

```diff
@@ -0,0 +1,92 @@
+import React, {useState} from 'react'
+import { Link } from 'gatsby'
+import styled from "styled-components"
+
+const StyledShortTextInput = styled.section`
+    width: 37.5rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    padding: 1em 1em 2em 1em;
+
+  div {
+    width: 23rem;
+    /* height: 15rem; */
+    border: 1px solid var(--dark-grey);
+    border-radius: 2.2px;
+    padding: .2rem 1rem;
+    /* margin: 1rem 1rem 2rem 1rem; */
+  }
+  div:active {
+    border: 2px solid var(--button-outline-blue);
+  }
+
+  div:focus-within {
+    border: 2px solid var(--button-outline-blue);
+  }
+
+  label {
+    font-weight: 300;
+    font-size: .8em;
+    color: var(--dark-grey);
+    display: block;
+    margin-bottom: .1rem;
+  }
+  
+  input {
+    margin-left: .1rem;
+    border: none;
+    resize: none;
+    font-weight: 400;
+    color: var(--tints-blizzard-black);
+  }
+  
+  input:focus {
+    outline: none;
+  }
+  
+  span {
+    width: 23rem;
+    line-height: 1.3em;
```

Comment:
It's actually recommended line-height be set with a unitless value. Check [this article](https://css-tricks.com/almanac/properties/l/line-height/) for more info but you can remove the em here.

Reply:
Got it